pkg/arm/v7/data/master-startup.sh

```diff
@@ -169,11 +169,10 @@ systemctl stop rsyslog.service
 
 # note: atomic-openshift-node crash loops until master is up
 systemctl enable atomic-openshift-node.service
-systemctl start atomic-openshift-node.service &
 {{ if .Config.SecurityPatchPackages }}
-needs-restarting --reboothint &>/dev/null || {
-  logger -t master-startup.sh "rebooting $(hostname) to complete ARO security updates"
-  shutdown --reboot now
-}
+logger -t master-startup.sh "scheduling $(hostname) reboot to complete ARO security updates"
+shutdown --reboot +5
```

Comment:
Is there a way to shrink this `5` to `2` or is this the right answer?  Just want to make sure this remains efficient.

Reply:
Just to start discussion: 
```
systemctl enable atomic-openshift-node.service
{{ if .Config.SecurityPatchPackages }}
needs-restarting --reboothint
if [ $? -eq 0 ]; then
     logger -t master-startup.sh "scheduling $(hostname) reboot to complete ARO security updates"
     shutdown --reboot now
else
    systemctl start atomic-openshift-node.service
fi
{{else}}
    systemctl start atomic-openshift-node.service
{{end}}
```
This way we don't even start node on the first update and we go via full cycle after reboot. This would prevent nodes going `Ready, NotReady, Ready`.

But my concert is for a hypothetical situation where `needs-restarting --reboothint` always returns `true` ? Which should not be a case.

Reply:
I did a few local test runs yesterday with values from 2 minutes to 5 minutes and all values except 5 minutes were flaky.